cranelift/codegen/src/isa/riscv64/inst.isle

```diff
@@ -1798,7 +1798,8 @@
   (let
     ((result WritableReg (temp_writable_reg out_type))
       (tmp WritableReg (temp_writable_reg $F64))
-      (_ Unit (emit (MInst.FcvtToInt is_sat result tmp rs is_signed in_type out_type))))
+      (rs2 Reg (clamp_fcvt_from_float in_type out_type rs is_signed))
```

Comment:
This seems to target the whole family of `fcvt_*` instructions, but the clamp here is really only needed for `fcvt_*_sat` which do the saturating conversions.